File: LogOpe.java
//Logical Operator
class LogOpe{
public static void main(String arg[]){
	
	int a=21,c=0;
	char b='t';
	if (a>0  &&  a%2==0) //Logical  and Operator
	System.out.println("Number is even natural number");
	else 
	System.out.println("Number is not even");

	if(b == 'a'|| b == 'e' || b =='i' || b =='o' || b =='u')   //Logical or operator
	System.out.println("vowel character");
	else
	System.out.println("constant ");

	if( c!=0 )				//not equal
	System.out.println("num is non zero");
	else
	System.out.println("num is zero");
}
}

File: LogOpe.cpp

#include <iostream>
using namespace std;
int main(){

	int a=21,c=0;
	char b='t';
	if (a>0  &&  a%2==0)		//logical and 
	cout<<"Number is even natural number"<<endl;
	else 
	cout<<"Number is not even"<<endl;

	if(b == 'a'|| b == 'e' || b =='i' || b =='o' || b =='u') //Logical or  
	cout<<"vowel character"<<endl;
	else
	cout<<"constant "<<endl;

	if( c!=0 )				//Not equal
	cout<<"num is non zero"<<endl;
	else
	cout<<"num is zero"<<endl;
}

